README.md

```diff
@@ -100,6 +100,7 @@
 ### Serialization
 
 - [vproto](https://github.com/emily33901/vproto) - Protobuf compiler and runtime in V.
+- [xml](https://github.com/radare/v-toxml) - XML Serialization library for V.
```

Comment:
I think it should be named `v-toxml` here instead of `xml`.

Reply:
Yes, that's more correct